visualization_scripts/environment.yml

```diff
@@ -0,0 +1,101 @@
+name: py2cyto
+channels:
+  - defaults
```

Comment:
We can specify the `conda-forge` channel here for igraph.  It will look like https://github.com/gitter-lab/t-cell-classification/blob/master/environment.yml

Reply:
I just added a minimal environment file to my visualization branch (sandbox for the visualization development) https://github.com/ajshedivy/tps/blob/visualization/visualization_scripts/minimal_env.yml 
When plugging this environment file into my workflow I had no issues. Will run further test cases

Reply:
To move this environment, first start in the branch for this pull request:
`git checkout visualizationPR`

Then checkout the new env file from the other branch (assumes you are currently in the base directory):
`git checkout visualization -- visualization_scripts/minimal_env.yml`

That will add the file to the `visualizationPR` branch, which you can then commit and push to the pull request.  Or you could delete `environment.yml` and `mv minimal_env.yml environment.yml`.